docs/README.md

```diff
@@ -101,6 +100,7 @@ When adding new articles, you MUST add them in both the `_articles/en` and `_art
 
 When updating the `_/data/sidebar.yml` file, titles must specify both the `en` and `ja` versions. If you are adding an new item in english simply copy the english value to the japanese value and it will be translated on the next localization pass.
 
-```
+## Github Pages
 
-```
+Previously the docs site was hosted on Github pages for the main SF VSCode extensions repository. This has been decommissioned in favor of the [doc site](https://developer.salesforce.com/tools/vscode).
+A javascript redirect has been put in place on the 404.html file to ensure we land on the documentation site when the github pages site is accessed.
```

Comment:
not: JavaScript should probably be UpperCamelCase (that's how it is on their official site https://www.javascript.com/)